doc/release-notes/5.10-release-notes.md

```diff
@@ -0,0 +1,315 @@
+
+The Search API now allows arbitrary metadata fields to be requested when displaying results from datasets. You can request all fields from metadata blocks or pick and choose certain fields.
+
+The new parameter is called `metadata_fields` and the Search API documentation contains details and examples: <https://guides.dataverse.org/en/5.10/api/search.html>
+
+### Solr 8 Upgrade
+
+The Dataverse Software now runs on Solr 8.11.1, the latest available stable release in the Solr 8.x series.
+
+### PostgreSQL Upgrade
+
+A PostgreSQL upgrade is not required for this release but is planned for the next release. See below for details.
+
+## Major Use Cases and Infrastructure Enhancements
+
+Newly-supported major use cases in this release include:
+
+- When creating/updating datasets, users can select from a set of standard licenses configured by the administrator or provide custom terms (if the installation is configured to allow them). (Issue #7440, PR #7920)
+- Users can get better feedback on tabular ingest errors and more information about size limits when uploading files. (Issue #8205, PR #8271)
+- Users can more easily download guestbook responses and learn how guestbooks work. (Issue #8244, PR #8402)
+- Search API users can specify additional metadata fields to be returned in the search results. (Issue #7863, PR #7942)
+- The "Preview" tab on the file page can now show restricted files. (Issue #8258, PR #8265)
+- Users wanting to upload files from GitHub to Dataverse can learn about a new GitHub Action called "Dataverse Uploader". (PR #8416)
+- Users requesting access to files now get feedback that it was successful. (Issue #7469, PR #8341)
+- Users may notice various accessibility improvements. (Issue #8321, PR #8322)
+- Guestbooks now support multi-line text area fields. (Issue #8288, PR #8291)
+- Guestbooks can better handle commas in responses. (Issue #8193, PR #8343)
+- Dataset editors can now deselect a guestbook. (Issue #2257, PR #8403)
+- Administrators with a large `actionlogrecord` table can read docs on archiving and then trimming it. (Issue #5916, PR #8292)
+- Administrators can list locks across all datasets. (PR #8445)
+- Administrators can run a version of Solr that doesn't include a version of log4j2 with serious known vulnerabilities. We trust that you have patched the version of Solr you are running now following the instructions that were sent out. An upgrade to the latest version is recommended for extra peace of mind. (PR #8415)
+- Administrators can run a version of the Dataverse Software that doesn't include any version of log4j (neither version 1 nor 2). log4j2 has never been included with the Dataverse Software. As of this version, neither does the Dataverse Software include log4j1, which had vulnerabilities that were much less severe than log4j2. We are not aware of any practical exploits that could be used to compromise the Dataverse Software based on its inclusion of log4j1 in previous releases. (PR #8377)
```

Comment:
Seems a little strong given that we do include reload4j.  Also - I was also working on #8475 yesterday which would mean log4j2 with patches would come back if we want to stay up to date with tika. I don't want to wordsmith in detail but was thinking just a statement that Dataverse does not include any vulnerable versions of log4j 1 or 2 might be enough without potentially giving people the idea that log4j is fully gone and/or won't come back going forward.

Reply:
@qqmyers @landreev as agreed in Slack, in 05bb1da147 I reverted to the original, simple statement:

`- Administrators can run a version of Dataverse that doesn't include a version of log4j with known vulnerabilities. (PR #8377)`